src/main/java/org/jabref/gui/collab/EntryChangeViewModel.java

```diff
@@ -55,6 +56,7 @@ public Node description() {
         header.getStyleClass().add("sectionHeader");
         container.getChildren().add(header);
         container.getChildren().add(mergePanel);
+        container.setMargin(mergePanel, new Insets(5, 5, 5, 5));
```

Comment:
This does not seem to work, as there appears to be no space on the left side (or 5 is too small)